cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,194 @@
+    consumer_key  = ansibleModule.params.get('consumer_key')
+
+    return ovh.Client(
+        endpoint=endpoint,
+        application_key=application_key,
+        application_secret=application_secret,
+        consumer_key=consumer_key
+    )
+
+def waitForNoTask(client, name):
+    while len(client.get('/ip/loadBalancing/{}/task'.format(name)))>0:
+        time.sleep(1)  # Delay for 1 sec
+    
+def main():
+    module = AnsibleModule(
+        argument_spec = dict(
+            name = dict(required=True),
+            backend = dict(required=True),
+            weight = dict(default='8'),
+            probe = dict(default='none', choices =['none', 'http', 'icmp' , 'oco']),
+            state = dict(default='present', choices=['present', 'absent']),
+            endpoint = dict(required=True),
+            application_key = dict(required=True),
+            application_secret = dict(required=True),
+            consumer_key = dict(required=True),
+        )
+    )
+    
+    if not HAS_OVH:
+        module.fail_json(msg='ovh-api python module is required to run this module ')
+
+    # Get parameters
+    name   = module.params.get('name')
+    state  = module.params.get('state')
+    backend  = module.params.get('backend')
+    weight  = long(module.params.get('weight'))
+    probe  = module.params.get('probe')
+
+    # Connect to OVH API
+    client = getOvhClient(module)
+
+    # Check that the load balancing exists
+    try :
+        loadBalancings = client.get('/ip/loadBalancing')
+    except APIError as apiError:
+        module.fail_json(msg='Unable to call OVH api for getting the list of loadBalancing, check application key, secret, consumerkey and parameters. Error returned by OVH api was : {}'.format(apiError))
+        
+    if not name in loadBalancings:
+        module.fail_json(msg='IP LoadBalancing {} does not exist'.format(name))
+
+    # Check that no task is pending before going on
+    waitForNoTask(client, name)
+
+    try :
+        backends = client.get('/ip/loadBalancing/{}/backend'.format(name))
+    except APIError as apiError:
+        module.fail_json(msg='Unable to call OVH api for getting the list of backends of the loadBalancing, check application key, secret, consumerkey and parameters. Error returned by OVH api was : {}'.format(apiError))
+    
+    backendExists = backend in backends
+    moduleChanged = False
+    if state=="absent" :
+        if backendExists :
+            # Remove backend
+            try :
+                client.delete('/ip/loadBalancing/{}/backend/{}'.format(name, backend))
+                waitForNoTask(client, name)
+            except :
+                module.fail_json(msg='Unable to call OVH api for deleting the backend, check application key, secret, consumerkey and parameters. Error returned by OVH api was : {}'.format(apiError))
+            moduleChanged = True
+        else :
+            moduleChanged = False
```

Comment:
moduleChanged is already False when you enter that branch, it can be removed (cf line 141)